### ISSUE TITLE
Add spin-trigger-command app to the tests

### ISSUE THREAD

**Issue by vdice** [spinkube/containerd-shim-spin#143]
Do we want to add an app using the [spin-trigger-command](https://github.com/fermyon/spin-trigger-command) plugin to the test suite?

**Comment by Mossaka** [comment]
Yes I had added an image in a branch to test Spin Command Trigger in a local branch. Will find it and make a PR
